stix_shifter_modules/qradar/stix_translation/results_translator.py

```diff
@@ -21,6 +21,34 @@ def translate_results(self, data_source, data):
       
       if result.get('flowdestinationpayload'):
         result['mime_type_flowdestinationpayload'] = 'application/octet-stream'
+      
+      if result.get('sourceip'):
+        if result['sourceip'] == '0.0.0.0':
+          result['sourceip'] = None
```

Comment:
can you use a transformer for this case instead of hard coding the values?

Reply:
Hi mdazam1942, these changes where already approved and merged into Develop by Danny. This is is for adding them into the Develop_4.x.